src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -507,12 +507,12 @@ class IntelliJPlugin implements Plugin<Project> {
         Zip zip = project.tasks.create(BUILD_PLUGIN_TASK_NAME, Zip).with {
             description = "Bundles the project as a distribution."
             group = GROUP_NAME
-            from({
-                [
-                        "${prepareSandboxTask.getDestinationDir()}/${prepareSandboxTask.getPluginName()}",
-                        jarSearchableOptionsTask.getDestinationDir()
-                ]
-            })
+            from { "${prepareSandboxTask.getDestinationDir()}/${prepareSandboxTask.getPluginName()}" }
+            from({ jarSearchableOptionsTask.getDestinationDir() }) {
```

Comment:
can't you use just `from { "${jarSearchableOptionsTask.getDestinationDir()}/${jarSearchableOptionsTask.getArchiveName()}"}`?

Reply:
Nope. The structure is all follows: `pluginName/lib/*.jar`. If I do this
the file is copied to `pluginName` but I need it in `pluginName/lib`. I'll
try it again tomorrow but I think it didn't work or maybe I just forgot
braces :)

Reply:
Confirm, your suggestion leads to `pluginName/searchableOptions.jar` instead of `pluginName/lib/searchableOptions.jar`.

Reply:
can you adjust `to` then?
it's just quite weird that we need to build complex include rule when actually we know the exact path of a file to copy

Reply:
Can we be sure that `${prepareSandboxTask.getDestinationDir()}/${prepareSandboxTask.getPluginName()}` always contains only `lib` directory? In this case, we can adjust `into` as it's used by both paths.

Reply:
yes, `lib` directory might be missing if you have no sources in the project. But in this case, searchable options will be also missing.